NetKAN/ContractPackRemoteTechLITE.netkan

```diff
@@ -0,0 +1,16 @@
+{
+    "spec_version": 1,
+    "license": "CC-BY-NC-SA-4.0",
+    "$kref": "#/ckan/kerbalstuff/938",
+    "identifier": "ContractPackRemoteTechLITE",
```

Comment:
The established convention for these types of mods would make this `ContractConfigurator-RemoteTechLite`.